app/http/controllers/user2.js

```diff
@@ -52,7 +52,7 @@ module.exports.createUser = function (User) {
       prefLocale: req.body.user.prefLocale,
       referrer: req.body.user.referrer,
       lastLoggedIn: new Date(),
-      teach: req.body.teach
+      client_id: req.body.oauth ? req.body.oauth.client_id : ""
```

Comment:
I think we should set this to undefined, not an empty string. Or maybe check with an `if` statement first?